scripts/gitlab/build_and_test.sh

```diff
@@ -66,7 +66,7 @@ if [[ -z ${hostconfig} ]]
 then
     # If no host config file was provided, we assume it was generated.
     # This means we are looking of a unique one in project dir.
-    hostconfigs=( $( ls "${project_dir}/"hc-*.cmake ) )
+    hostconfigs=( $( ls "${project_dir}/tpl/RAJA/"hc-*.cmake ) )
```

Comment:
@kab163 I don't like the change I made to this line. The host-config files are generated in the ./tpl/RAJA directory, but the build step expects them to be in the top-level RAJAPerf directory. Alternatively, we could copy them from tpl/RAJA to RAJAPerf directory. Any thoughts on this, or other alternatives?

Reply:
Ah, I see what you mean. I just saw that in the Gitlab CI output - `Found host config file: /g/g0/belcher6/.jacamar-ci/builds/jeegSUGP/000/gitlab/radiuss/RAJAPerf/tpl/RAJA/hc-corona-toss_4_x86_64_ib-clang@13.0.0-er4dwqli75iqdevmeftcywtinq5jljkf.cmake` - that is a bit odd. IMHO, I think it would be fine to copy it from tpl/RAJA to RAJAPerf...

Reply:
Why did you update it initially? (Could it just be left in RAJAPerf?)

Reply:
Update what?

Reply:
Sorry, I meant update as in adding `tpl/RAJA` right there.

Reply:
Because the host-config files are generated in the tpl/RAJA directory. Not sure how to get it to generate in the top-level RAJAPerf directory where it is expected to be to build.